.gitignore

```diff
@@ -4,8 +4,10 @@
 target/
 /var
 /*/var/
+/presto-product-tests/**/var/
 pom.xml.versionsBackup
```

Comment:
Conf directory shouldn't be excluded. It contains configurations for running product tests in docker. See `run_in_docker.sh`.